packages/amqp/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "@idearium/amqp",
+    "version": "1.0.0-beta.4",
+    "description": "A library for working with AMQP.",
+    "main": "index.js",
+    "repository": "https://github.com/idearium/idearium-lib",
+    "author": "Scott Mebberson <scott@idearium.io>",
+    "license": "MIT",
+    "dependencies": {
+        "@idearium/log": "2.1.0-beta.1",
+        "@idearium/promise-all-settled": "1.0.0",
+        "@idearium/safe-promise": "1.0.0",
+        "amqplib": "0.10.3"
+    },
+    "scripts": {
+        "test": "jest"
+    },
+    "engines": {
+        "node": ">=12"
```

Comment:
I think we should set this to 18 (current LTS) or at the very least 14 which is in maintenance for another month.

Reply:
@allanchau, I've updated this to `>=16`.